### ISSUE TITLE
IMDB TV: Won't scrape from episode 51 onwards

### ISSUE THREAD

**Issue by ScoobyD00M** [Komet/MediaElch#1774]
**Describe the bug**
If you scrape a series with a large number of episodes per season with IMDB TV, no more information is loaded for every episode with a episode number higher than 50

**To Reproduce**
1. Find a TV series with more than 50 episodes per season (e.g. Fullmetal Alchemist: Brotherhood, Batman TAS or some game shows/daily soaps)
2. Load informationen with IMDB TV for any episode from S01E51 onwards

**Expected behavior**
Load information for all episodes of every season

**MediaElch Version:**
2.10.6 (stable)

**Operating System:**
Windows 11

**Comment by bugwelle** [comment]
Oh, thanks for reporting! :)

**Comment by kolbdog323** [comment]
Will you ever get chance to look at this bug this person posted a few months ago sicne be nice since some shows on imdb have more then 50 episodes in a season.

**Comment by bugwelle** [comment]
Yes and no. The issue is that IMDb does not provide any pagination, it seems. It loads the next episodes via the "see more link" with JavaScript. If you look at https://www.imdb.com/title/tt1355642/episodes/?season=1 , it does not contain all episodes. I can't fake that API request, as it uses some token that's set per-request.

As of now, this issue won't be resolved. If anyone finds a link to a "Page 2" for the link above, please let me know.
